db/seeds/development/users.seeds.rb

```diff
@@ -12,7 +12,8 @@ def create_random_user(i, preferred_party_id, willing_party_id)
     email: "#{firstname.downcase}.#{i}-#{preferred_party_id}-#{willing_party_id}@example.com",
     constituency_ons_id: ons_id,
     preferred_party_id: preferred_party_id,
-    willing_party_id: willing_party_id
+    willing_party_id: willing_party_id,
+    password: "123456"
```

Comment:
I think we need this now passwords are required. Without this we don't get any users when we seed the DB.